usr/share/lib/img_proof/tests/SLES/test_sles_grow_root.py

```diff
@@ -56,15 +61,15 @@ def test_sles_grow_root(host):
     if var_part and var_part != root_part:
         # Some images have a separate /var partition that is >= 1G
         result = host.run(
-            'df -BG {part} | sed 1D'.format(part=var_part)
+            'df -B{unit} {part} | sed 1D'.format(unit=unit, part=var_part)
         ).stdout.strip()
 
         var_size = shlex.split(result)[1]  # Filesystem 1G-blocks
-        var_size = int(var_size.replace('G', ''))
+        var_size = float(var_size.replace(unit, ''))
     else:
         var_size = 0
 
     total_size = root_size + boot_size + var_size
 
     # Rounding can lead to small discrepancies
-    assert total_size in (disk_size - 1, disk_size, disk_size + 1)
+    assert (disk_size - 1) <= total_size <= (disk_size + 1)
```

Comment:
This chaining syntax is to he best of my knowledge Python interpreter version specific. I think it is safer to use ``assert (((disk_size - 1) <= total_size) and (total_size <= (disk_size + 1)))```